slick-testkit/src/main/scala/com/typesafe/slick/testkit/tests/InsertTest.scala

```diff
@@ -230,4 +230,90 @@ class InsertTest extends AsyncTest[JdbcTestDB] {
       }
     } yield ()).withPinnedSession
   }
+
+  // TODO DerbyProfile and PostgresProfile seems to have a problem in this case. check #2206 and #2207
+  def testInsertOrUpdateWithPrimaryKeyOnly: DBIOAction[Unit, NoStream, Effect.All] = if (!tdb.confName.matches("(derby|postgres).*")) {
```

Comment:
#1627, but Postgres and Derby is not supported for now.

Reply:
#2206 #2207